test/test_codegen.py

```diff
@@ -29,16 +29,123 @@
 import pyrometheus as pyro
 
 
-def test_sandiego():
+def test_get_pressure():
```

Comment:
Might it make sense to parametrize this so that it gets done with more than one mechanism?

Reply:
Agreed. Panned as one of next steps.